src/Schema/SqliteSchemaManager.php

```diff
@@ -379,6 +379,12 @@ protected function _getPortableTableColumnDefinition($tableColumn)
             $default = null;
         }
 
+        // https://github.com/doctrine/dbal/blob/3.8.4/src/Platforms/SqlitePlatform.php#L271-L274
+        // https://dbfiddle.uk/yyXvHV-6
+        if (strtolower($type) === 'integer' && ($tableColumn['pk'] !== 0 && $tableColumn['pk'] !== '0')) {
```

Comment:
I don't understand why `$tableColumn['pk'] !== 0 && $tableColumn['pk'] !== '0'` should mean this is an auto increment column. Couldn't it be an integer column that is part of the primary key?

Reply:
I coded the "auto increment" detection based on existing https://github.com/doctrine/dbal/blob/b6a65d9ef8/src/Schema/SqliteSchemaManager.php#L295-L319. Is that right?

Reply:
I also noticed that code block, but I don't know why it is right (if it is).